src/main/java/com/ebay/myriad/scheduler/TaskFactory.java

```diff
@@ -73,10 +143,10 @@ public TaskInfo createTask(Offer offer, TaskID taskId, NodeTask nodeTask) {
             nmTaskConfig.setAdvertisableCpus(profile.getCpus());
             nmTaskConfig.setAdvertisableMem(profile.getMemory());
             NodeManagerConfiguration nodeManagerConfiguration = this.cfg.getNodeManagerConfiguration();
-            nmTaskConfig.setUser(nodeManagerConfiguration.getUser().orNull());
             nmTaskConfig.setJvmOpts(nodeManagerConfiguration.getJvmOpts().orNull());
             nmTaskConfig.setCgroups(nodeManagerConfiguration.getCgroups().or(Boolean.FALSE));
             nmTaskConfig.setYarnEnvironment(cfg.getYarnEnvironment());
+            nmTaskConfig.setRemoteDistribution(cfg.getMyriadExecutorConfiguration().getNodeManagerUri().isPresent());
```

Comment:
Sure, we can defer on updating config of a preinstalled Hadoop. I just wanted to throw the idea out there so we don't code ourselves into a corner.
I still don't think we need a separate remoteDistribution boolean, since each URI will already specify if it's a file:// or absolute/relative path. Even for a localDistribution, couldn't the hadoop and executor binaries live separately?